packages/component-library/src/components/navigation/mt-link/mt-link.vue

```diff
@@ -6,6 +6,7 @@
       `mt-link--${variant}`,
       {
         'mt-link--disabled': disabled,
+        [`mt-link--${type}`]: type !== undefined,
       },
     ]"
     :href="disabled ? undefined : to"
```

Comment:
The link component does not add a `target=_blank` to external links. Should we add it?

Reply:
I would leave it like it is right now. With the default browser behaviour we allow the user to choose if they want to open the link in the same tab or to press CTRL to open it in a new tab. 

If we use `taget="_blank"` we force the user to open the tab in a new page.

[More reading](https://css-tricks.com/use-target_blank/)